site/realm/data_sources/mongodb-atlas/customData/users/schema.json

```diff
@@ -17,6 +17,15 @@
         },
         "userId": {
             "bsonType": "string"
+        },
+        "adminData": {
+            "bsonType": "object",
+            "properties": {
+              "email": {"bsonType": "string"},
+              "disabled": {"bsonType": "bool"},
+              "creationDate": {"bsonType": "string"},
+              "lastAuthenticationDate": {"bsonType": "string"}
+            }
         }
```

Comment:
Is this needed for this feature?

Reply:
When I reset it to staging, the deploy preview on my fork breaks with:

```
7:09:22 PM:   Running:  integration/admin.cy.js                                                        (3 of 62)
7:09:25 PM: 
7:09:25 PM:   Admin
7:09:26 PM:     ✓ Should show not enough permissions message (1127ms)
7:09:33 PM:     (Attempt 1 of 3) Should display a list of users, their roles abnd allow edition
7:09:34 PM:     (Attempt 2 of 3) Should display a list of users, their roles abnd allow edition
7:09:35 PM:     1) Should display a list of users, their roles abnd allow edition
7:09:39 PM:     ✓ Should display New Incident button (4582ms)
7:09:39 PM:   2 passing (14s)
7:09:39 PM:   1 failing
7:09:39 PM:   1) Admin
7:09:39 PM:        Should display a list of users, their roles abnd allow edition:
7:09:39 PM:      TypeError: Cannot read properties of undefined (reading "userId")
7:09:39 PM:       at Context.eval (webpack://gatsby-gitbook-aiid/./cypress/e2e/integration/admin.cy.js:65:54)
7:09:39 PM: 
7:09:39 PM:   (Results)
7:09:39 PM:   ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
7:09:39 PM:   │ Tests:        3                                                                                │
7:09:39 PM:   │ Passing:      2                                                                                │
7:09:39 PM:   │ Failing:      1                                                                                │
7:09:39 PM:   │ Pending:      0                                                                                │
7:09:39 PM:   │ Skipped:      0                                                                                │
7:09:39 PM:   │ Screenshots:  0                                                                                │
7:09:39 PM:   │ Video:        false                                                                            │
7:09:39 PM:   │ Duration:     14 seconds                                                                       │
7:09:39 PM:   │ Spec Ran:     integration/admin.cy.js                                                          │
7:09:39 PM:   └────────────────────────────────────────────────────────────────────────────────────────────────┘
```

I don't see that it has anything to do with this pull request, but I think it's needed for some reason.